create-bitcoin-wallet.md

```diff
@@ -0,0 +1,82 @@
+# How to create a Bitcoin wallet
+
+Creating a Bitcoin wallet on BlueWallet is easy.
+
+You can either click on the "Add now" button or the "+" icon in the top right corner
+
+![](./images/bluewallet-first-page.png)
+
+That will take you to the "add wallet" screen.
+
+## Adding a new wallet
+
+In the "add wallet" screen, you can choose between a Bitcoin wallet and a Lightning wallet.
+
+![](./images/add-bitcoin-wallet.png)
+
+For Bitcoin, choose Bitcoin. (If you want to create a Lightning wallet, see [this guide](./create-lightning-wallet.md).)
+
+You may also name your wallet, or leave the default name.
+
+<div class="side-note">
+
+### Advanced options
+
+If you have "Advanced mode" enabled in general settings,
+you'll see more options for creating the wallet.
+
+![](./images/add-bitcoin-wallet-with-advanced-options.png)
+
+#### Wallet type
+
+The first group of options is for the wallet type:
```

Comment:
I'd like to add more details about the different types. Why would the user choose one type over the other? Are there trade offs? Or is it for compatibility with other wallets?